db/migrate/20180812162710_create_status_stats.rb

```diff
@@ -0,0 +1,12 @@
+class CreateStatusStats < ActiveRecord::Migration[5.2]
+  def change
+    create_table :status_stats do |t|
+      t.belongs_to :status, foreign_key: { on_delete: :cascade }, index: { unique: true }
```

Comment:
I think that it is better to set `null: false`. Also, it seems that foreign_key is not reflected in `db/schema.rb`.